tests/end2end/features/misc.feature

```diff
@@ -436,7 +436,7 @@ Feature: Various utility commands.
     Scenario: Clicking an element with unknown ID
```

Comment:
You're probably already aware of this, but just so we don't forget - please also add some tests for the CSS selector.

Reply:
Yeah, I will definitely do that.